onnxruntime/core/providers/cuda/tensor/slice.cc

```diff
@@ -7,61 +7,59 @@
-                         InputMemoryType<OrtMemTypeCPUInput>(2).                          \
-                         InputMemoryType<OrtMemTypeCPUInput>(3).                          \
-                         InputMemoryType<OrtMemTypeCPUInput>(4).                          \
-                         TypeConstraint("T",    DataTypeImpl::AllFixedSizeTensorTypes()). \
-                         TypeConstraint("Tind", DataTypeImpl::GetTensorType<TIND>()),     \
-      Slice<TIND,true>);
-
-REGISTER_V10_TYPED_SLICE(int32_t) 
+#define REGISTER_V10_TYPED_SLICE(TIND)                                                                                                                                                                                                                                                                         \
+  ONNX_OPERATOR_TYPED_KERNEL_EX(                                                                                                                                                                                                                                                                               \
+      Slice,                                                                                                                                                                                                                                                                                                   \
+      kOnnxDomain,                                                                                                                                                                                                                                                                                             \
+      10,                                                                                                                                                                                                                                                                                                      \
+      TIND,                                                                                                                                                                                                                                                                                                    \
+      kCudaExecutionProvider,                                                                                                                                                                                                                                                                                  \
+      KernelDefBuilder().InputMemoryType<OrtMemTypeCPUInput>(1).InputMemoryType<OrtMemTypeCPUInput>(2).InputMemoryType<OrtMemTypeCPUInput>(3).InputMemoryType<OrtMemTypeCPUInput>(4).TypeConstraint("T", DataTypeImpl::AllFixedSizeTensorTypes()).TypeConstraint("Tind", DataTypeImpl::GetTensorType<TIND>()), \
```

Comment:
>KernelDefBuilder().InputMemoryType<OrtMemTypeCPUInput>(1).InputMemoryType<OrtMemTypeCPUInput>(2).InputMemoryType<OrtMemTypeCPUInput>(3).InputMemoryType<OrtMemTypeCPUInput>(4).TypeConstraint("T", DataTypeImpl::AllFixedSizeTensorTypes()).TypeConstraint("Tind", DataTypeImpl::GetTensorType<TIND>()), \ [](start = 6, length = 298)

don't you think it's too long for a line?

Reply:
Yes. Didn't notice the auto formatting by clang-format. Fixed.